internal/entitlement/metered/utils_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/stretchr/testify/assert"
 
 	"github.com/openmeterio/openmeter/internal/credit"
-	credit_postgres_adapter "github.com/openmeterio/openmeter/internal/credit/postgresadapter"
+	credit_postgres_adapter "github.com/openmeterio/openmeter/internal/credit/postgresdriver"
```

Comment:
What does this package contain? Isn't it an implementation of an interface from `credit`? If so, adapter is the correct name.

Also, for now we agreed to put everything under a single package (ie. single adapter and driver packages).